core/Utilities/GenericUtilities.cs

```diff
@@ -257,6 +291,34 @@ private static void BuildTreeNodes(CmfPackage pkg, TreeNode node)
                 }
             }
         }
+        
+        private static void BuildTreeNodes(CmfPackageV1 pkg, TreeNode node)
```

Comment:
Can't this accept an `IHasTreeNodes`? That way we could avoid duplicating this code in BuildTree.

Reply:
It could, but the point is to remove the old methods, so I'd leave the signature as explicit as possible.